packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -57,7 +200,101 @@ class FocusNode extends ChangeNotifier {
   /// ```
   ///
   /// This object notifies its listeners whenever this value changes.
```

Comment:
maybe add a link to [Listenable] or [addListener] here to make it clearer what that means.

Reply:
OK, done, and I added a _See also_ for `Focus.hasFocus` too (née `Focus.at`).